stable/artifactory-ha/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
+- jainishshah17
+- eldada
+- rimusz
+reviewers:
+- unguiculus
+- prydonius
+- mattfarina
```

Comment:
You you also please remove us from reviewers since the automation will automatically assign us if listed here.